content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -108,15 +50,15 @@ ENV['DD_VERSION'] = '2.5.17'
 logger = Logger.new(STDOUT)
 logger.progname = 'my_app'
 logger.formatter  = proc do |severity, datetime, progname, msg|
-  "[#{datetime}][#{progname}][#{severity}][#{Datadog.tracer.active_correlation}] #{msg}\n"
+  "[#{datetime}][#{progname}][#{severity}][#{Datadog::Tracing.log_correlation}] #{msg}\n"
 end
```

Comment:
It is correct, because a `#log_correlation` returns a string: https://github.com/DataDog/dd-trace-rb/blob/32fc71b7765a2578bdc8d9fb749b3ced9821543b/lib/datadog/tracing.rb#L95-L97

A formatted correlation string is in the vast majority of the use cases what users want, so we added this API for that.